src/core/templates-discovering.ts

```diff
@@ -95,10 +98,10 @@ function mountTemplate(
         name,
         inject: ["eventBus"],
         created(this: any & IEventBusHolder) {
-            this.eventBus.on("updated", updatedHandler.bind(this));
+            this.eventBus.add(updatedHandler.bind(this));
         },
         unmounted() {
-            this.eventBus.off("updated", updatedHandler);
+            this.eventBus.remove(updatedHandler);
```

Comment:
Trying to unsubscribe `updatedHandler`, though `updatedHandler.bind(this)` was used for subscription. These are different objects.